readme.md

```diff
@@ -25,6 +25,19 @@ console.log(element.id);
 
 Returns a promise for a matching element.
 
+### observeReadyElements(selector, options?)
+
+Returns an async iterator which yields with each new matching element. Useful to for example, url shorten `<a>` tags as they are added or to click buttons when they are clickable.
```

Comment:
```suggestion
Returns an async iterator which yields with each new matching element. Useful to, for example, shorten the URLs of `<a>` tags as they are added or to click buttons when they are clickable.
```

Reply:
Neither of these example use-cases are very common. I would try to come up with a more common use-case.

Reply:
Note to self:

> Useful in userscripts to modify elements when they are added or to wait for a specific amount of elements to load in